pkg/spec/populators.go

```diff
@@ -150,6 +167,93 @@ func populateEnvFrom(c Container, cms []ConfigMapMod, secrets []SecretMod) (Cont
 	return c, nil
 }
 
+// Parse the string the get the port, targetPort and protocol
+// information, and then return the resulting ServicePort object
+func parsePortMapping(pm string) (*api_v1.ServicePort, error) {
```

Comment:
the placement of this function is wrong can, you place it in `resources.go` or `util.go` ? Because this is file of all the `populators` !

Reply:
this function is just parsing, and not populating so logically it should reside somewhere but not with all other populatiors.

Reply:
ack, moved to resources.go